server/apps/archive/takes_package_service.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8; -*-
```

Comment:
You mean to move the archive_composite.py and takes_package_service.py to another folder module?